### PR TITLE
Platform support updates for 23.02.

### PR DESCRIPTION
**Pull request by bdice**
This PR updates our release selector for 23.02. @charlesbluca noticed that our Python versions have been incorrect for nightly versions of 23.02 (Python 3.9 has been dropped but has been showing on the release selector for nightly users).

@sevagh Can you please review the pip changes?

I would appreciate a critical eye from other reviewers as well. 🙏

### CODE REVIEW

File: start.md
Comment (raydouglass) [inline start.md:122]: We are still supporting CUDA 11.5, so I think you leave 11.5 and add 11.8 here.

File: _includes/selector.html
Comment (raydouglass) [inline _includes/selector.html:142]: Should leave 11.5 in and add 11.8

File: start.md
Comment (bdice) [reply start.md:122]: Good call. I always forget this because we dropped 11.5 from our PR CI. I think it’s still tested in nightlies?

**Review by raydouglass — CHANGES_REQUESTED** [review]
Requesting changes to block merging until the release is complete

**Review by vyasr — APPROVED** [review]
FWIW everything looks right to me. Only question is why in the WSL section the recommendations include Ubuntu 20.04 and 22.04, whereas before 18.04 wasn't included. Were there actual limitations with 18.04, was it not available via WSL2, or should we be continuing to recommend only one (the newest) Ubuntu version?

File: start.md
Comment (bdice) [reply start.md:123]: Fixed in d7fb44346379c3542d2df53566c62e631a11ebf7.

**Comment by bdice** [comment]
> FWIW everything looks right to me. Only question is why in the WSL section the recommendations include Ubuntu 20.04 and 22.04, whereas before 18.04 wasn't included. Were there actual limitations with 18.04, was it not available via WSL2, or should we be continuing to recommend only one (the newest) Ubuntu version?

I think we only tested on 20.04 in the pre-GA phase, and now we've dropped 18.04 so it's irrelevant. But now that we've hit "GA" quality releases and added 22.04 support, I didn't know if I should _replace_ 20.04 with 22.04 (which I run regularly and know to work) or just list both. I am fine with either approach and would defer to the opinions of @aravenel @raydouglass @taureandyernv @jarmak-nv. Should we only list the latest Ubuntu version in our WSL instructions?

File: _includes/selector.html
Comment (bdice) [inline _includes/selector.html:291]: @taureandyernv Where does this magic value of $2^{26}$ come from? Is it just "a large number" or does it mean something?

**Comment by jarmak-nv** [comment]
@bdice re: WSL Ubuntu Version

I think we should be good to just say use the latest ubuntu. It was _very slightly_ awkward before with Ubuntu 22.04 including Python 3.10 but now that we're on 3.10 it's aligned well. Constraining our recommendation reduces expected support as well which is not a bad thing for us.

**Comment by bdice** [comment]
> @bdice re: WSL Ubuntu Version
> 
> I think we should be good to just say use the latest ubuntu. It was _very slightly_ awkward before with Ubuntu 22.04 including Python 3.10 but now that we're on 3.10 it's aligned well. Constraining our recommendation reduces expected support as well which is not a bad thing for us.

Agreed. Done in 46f77656471292d083a6bc93b9ae437bb4bdc828.

**Review by beckernick — APPROVED** [review]
This LGTM pending @bdice 's question about the magic number for ulimit and @jarmak-nv 's PR https://github.com/rapidsai/rapids.ai/pull/295

**Review by ajschmidt8 — APPROVED** [review]

**Review by raydouglass — APPROVED** [review]

**Pull request merged**
